### ISSUE TITLE
GET recipes does not tell the max number of pages

### ISSUE THREAD

**Issue by udeyrishi** [SmartFoodTracker/backend#4]
Instead of making the response an array, could you make the response a JSON like:

```json
{ 
    "result": [ ... ], 
    "numPages": 10, 
    "pageNum": 1
}
```

**Comment by ahoskins** [comment]
new format is

```
{ 
    "data": [ ... ], 
    "totalPages": 10, 
    "page": 1
}
```

**Comment by udeyrishi** [comment]
@ahoskins Missed to update the documentation: https://smartfoodtracker.github.io/backend/#api-Recipes-GetRecipes

**Comment by ahoskins** [comment]
closed in a6394ba612136b78e254b128faa547590f385100
